opencti-platform/opencti-front/src/private/components/data/Management.tsx

```diff
@@ -0,0 +1,230 @@
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import DataTable from '../../../components/dataGrid/DataTable';
+import { UsePreloadedPaginationFragment } from '../../../utils/hooks/usePreloadedPaginationFragment';
+import useAuth from '../../../utils/hooks/useAuth';
+import { emptyFilterGroup, useBuildEntityTypeBasedFilterContext, useGetDefaultFilterObject } from '../../../utils/filters/filtersUtils';
+
+const LOCAL_STORAGE_KEY = 'restrictedEntities';
+
+export const managementDefinitionsLinesPaginationQuery = graphql`
+  query ManagementDefinitionsLinesPaginationQuery(
+    $search: String
+    $count: Int!
+    $cursor: ID
+    $orderBy: StixCoreObjectsOrdering
+    $orderMode: OrderingMode
+    $filters: FilterGroup
+    ) {
+      ...ManagementDefinitionsLines_data
+      @arguments(
+        search: $search
+        count: $count
+        cursor: $cursor
+        orderBy: $orderBy
+        orderMode: $orderMode
+        filters: $filters
+      )
+    }
+`;
+
+const managementDefinitionLineFragment = graphql`
+  fragment ManagementDefinitionsLine_node on StixCoreObject{
+    id
+    entity_type
+    created_at
+    updated_at
+    ... on StixObject {
+      representative {
+        main
+        secondary
+      }
+    }
+    createdBy {
+      ... on Identity {
+          name
+      }
+    }
+    objectMarking {
+      id
+      definition_type
+      definition
+      x_opencti_order
+      x_opencti_color
+      created
+      modified
+    }
+      objectLabel {
+        id
+        value
+        color
+      }
+    creators {
+      id
+      name
+    }
+  }
+`;
+
+const managementDefinitionsLinesFragment = graphql`
+  fragment ManagementDefinitionsLines_data on Query
+  @argumentDefinitions(
+    types: { type: "[String]" }
+    search: { type: "String" }
+    count: { type: "Int", defaultValue: 25 }
+    cursor: { type: "ID" }
+    orderBy: {
+      type: "StixCoreObjectsOrdering"
+      defaultValue: entity_type
+    }
+    orderMode: { type: "OrderingMode", defaultValue: asc }
+    filters: { type: "FilterGroup" }
+  )
+  @refetchable(queryName: "ManagementDefinitionsLinesRefetchQuery") {
+    stixCoreObjectsRestricted(
+      types: $types
+      search: $search
+      first: $count
+      after: $cursor
+      orderBy: $orderBy
+      orderMode: $orderMode
+      filters: $filters
+    ) 
+    @connection(key: "Pagination_stixCoreObjectsRestricted") {
+      edges {
+        node {
+          id
+          entity_type
+          ...ManagementDefinitionsLine_node
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+        globalCount
+      }
+    }
+  }
+`;
+
+const Management = () => {
+  const { t_i18n } = useFormatter();
+
+  const {
+    platformModuleHelpers: { isRuntimeFieldEnable },
+  } = useAuth();
+  const isRuntimeSort = isRuntimeFieldEnable() ?? false;
```

Comment:
`isRuntimeFieldEnable` seems to always return a **bool** nullish coalescing operator might be useless.
```suggestion
  const isRuntimeSort = isRuntimeFieldEnable();
```